src/pages/Admin/Venue/DetailsValidationSchema.ts

```diff
@@ -108,7 +108,9 @@ export const validationSchema = Yup.object()
     attendeesTitle: Yup.string().notRequired().default("Guests"),
 
     logoImageUrl: urlIfNoFileValidation("logoImageFile"),
+    // TODO: make not required
     description: Yup.string().required("Description required"),
+    // TODO: make not required
```

Comment:
Already done in https://github.com/sparkletown/sparkle/pull/2009

Reply:
@dmitrymarchuk thank you a lot